### ISSUE TITLE
Warning when missing paramers with StrSubstNo

### ISSUE THREAD

**Issue by guidorobben** [ALCops/Analyzers#194]
I thought that  when the parameters like %1 where missing in the StrSubstNo function, it would produce a warning. I am not sure if this was a check of MS or a LinterCop one. But currently it is not checked anymore.

```
 trigger OnRun()
 var
        TestMsg: Label 'It should give an error for this: %1', Comment = '%1=Any text that needs to be tested';
        CompleteText: Text;
 begin
        CompleteText := StrSubstNo(TestMsg); // Missing %1 Param
 end;
```

It should have given a warning that %1 is missing.

**Comment by Arthurvdv** [comment]
I think you've found a issue on the [CodeCop AA0131 - String parameters must match placeholders](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/analyzers/codecop-aa0131).

I've created a new rule PC0034 that will fill in the gaps, to cover the missing scenario's (see details on the linked PR).

Thanks for creating an issue for this

**Comment by Arthurvdv** [comment]
@guidorobben, with the release of version `v0.8.0` of ALCops, this new rule is now included in the official release.

I’m going to close this issue now, but if you have any feedback or run into bugs/false positives, feel free to reopen it or create a new issue.
